Bug: test_vxlan_crm setup failed due to missing backup_and_restore_config_db_on_duts fixture

### Is it platform specific

generic

### Importance or Severity

High

### Description of the bug

See below error when running test_vxlan_crm.py
```
_______________________________________________________________`
Test_VxLAN_Crm.test_crm_128_group_members[v6_in_v6]
_______________________________________________________________
file /data/tests/vxlan/test_vxlan_crm.py, line 213                                                                                                                                                  
      def test_crm_128_group_members(self, setUp, encap_type):                                                                                                                                      
file /data/tests/vxlan/test_vxlan_ecmp.py, line 139                                                                                                                                                 
  @pytest.fixture(name="setUp", scope="module")                                                                                                                                                     
  def fixture_setUp(duthosts,                                                                                                                                                                       
E       fixture 'backup_and_restore_config_db_on_duts' not found                                                                                                                                    
```
test_vxlan_crm.py uses setUp in test_vxlan_ecmp.py, which introduces new dependency on backup_and_restore_config_db_on_duts in PR  https://github.com/sonic-net/sonic-mgmt/pull/17767. 

### Steps to Reproduce

Run tests in test_vxlan_crm.py

### Actual Behavior and Expected Behavior

Setup should not fail

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/vxlan/test_vxlan_crm.py
+++ b/tests/vxlan/test_vxlan_crm.py
@@ -4,6 +4,7 @@ import ipaddress
 from functools import reduce
 
 from tests.common.helpers.assertions import pytest_assert
+from tests.common.fixtures.duthost_utils import backup_and_restore_config_db_on_duts    # noqa F401
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory     # noqa: F401
 from tests.common.vxlan_ecmp_utils import Ecmp_Utils
 from tests.vxlan.test_vxlan_ecmp import (   # noqa: F401

